firecrown/likelihood/gauss_family/statistic/source/weak_lensing.py

```diff
@@ -49,43 +49,33 @@ class MultiplicativeShearBias(WeakLensingSystematic):
 
     This systematic adjusts the `scale_` of a source by `(1 + m)`.
 
-    Parameters
-    ----------
-    mult_bias : str
-       The name of the multiplicative bias parameter.
     """
 
-    params_names = ["mult_bias"]
-    m: float
-
     def __init__(self, sacc_tracer: str):
-        """Create a MultipliciativeShearBias object that uses the named tracer.
+        """Create a MultiplicativeShearBias object that uses the named tracer.
         Parameters
         ----------
         sacc_tracer : The name of the multiplicative bias parameter.
         """
         super().__init__()
 
+        self.mult_bias = parameters.create()
         self.sacc_tracer = sacc_tracer
 
     @final
     def _update(self, params: ParamsMap):
```

Comment:
The `_update`, `_reset`, `_required_parameters` methods (and `_get_derived_parameters` to a lesser degree) add a lot of boiler plate but in most cases don't actually do anything any more, as shown by the many `"This implementation has nothing to do."`
I'd like to suggest removing the `@abstractmethod` for these in `Updatable`, or at least move the default implementations to places like `Systematic` and `Source`. I imagine the most common contributions to firecrown will be in the form of new systematics by users that neither understand nor care about the very neat internals you've come up with. I believe that allowing users to write a simple `Systematics` class with just an `__init__` and `apply` method makes for a more welcoming environment and readable code than having to copy in a bunch of empty `_*` methods that most users won't ever need to change. 

The existence of these methods should be documented for advanced users but even knowing what these methods do, I personally struggle to come up with a use case right now for anything but `_get_derived_parameters`.